docs/codeql/codeql-workshops/cpp/codeql-dataflow-sql-injection/codeql-dataflow-sql-injection.code-workspace

```diff
@@ -0,0 +1,17 @@
+{
+	"folders": [
+		{
+			"path": "."
+		},
+		{
+			"path": "../../../vmsync/ql"
```

Comment:
I think these paths are unique to your local setup. Could you either remove the workspace files (just ask users to open the repo and then import the database), or edit them so they are portable?

Reply:
done, now these paths are all just relative to the full repo